react-app/src/components/users/UserDetail.tsx

```diff
@@ -96,6 +98,11 @@ const UserDetail = ({ onClose }: IUserDetail) => {
     mode: 'onBlur',
   });
 
+  let canEdit = false;
+  if (user.keycloak.hasRoles([Roles.ADMIN])) {
+    canEdit = true;
+  }
+
```

Comment:
Could probably condense this.

```suggestion
  const canEdit = user.keycloak.hasRoles([Roles.ADMIN]);
```